src/Bowling.js

```diff
@@ -0,0 +1,57 @@
+var Bowling = function() {
+  this.bowlingFrame = 1;
+  this.frames = {};
+  for (i = 1; i <= 9; i++) {
+    this.frames[i] = new Frame(0);
+  };
+  this.frames[10] = new Frame(2)
+  this.bowls = 0;
+};
+
+Bowling.prototype.bowl = function(score) {
+  if(this.bowlingFrame > 10) {
+    throw new Error('it is the end of the game');
+  };
+  if(this.bowlingFrame <= 10) {
+    this.frames[this.bowlingFrame].bowl(score);
+    this.bowls = score;
+  };
+  if(this.bowlingFrame > 1){
+    this.bonus(1);
+    this.bonus(2);
+  };
+  if(this.nextFrame()) {
+    this.bowlingFrame += 1;
+  };
+};
```

Comment:
?! Mysterious function - I really don't know what this does on first sight, and all I wanted it to do is to bowl